src/core_modules/capture-core/components/WidgetEnrollment/WidgetEnrollment.component.js

```diff
@@ -82,7 +84,7 @@ export const WidgetEnrollmentPlain = ({
                 {loading && <LoadingMaskElementCenter />}
                 {!error && !loading && (
                     <div className={classes.enrollment}>
-                        <div data-test="widget-enrollment-status">
+                        <div className={classes.statuses} data-test="widget-enrollment-status">
```

Comment:
Putting layout styles on a container, rather than on the individual status objects.